waku/v2/node/keepalive.go

```diff
@@ -36,23 +41,47 @@ func (w *WakuNode) startKeepAlive(ctx context.Context, t time.Duration) {
 		select {
 		case <-ticker.C:
 			difference := w.timesource.Now().UnixNano() - lastTimeExecuted.UnixNano()
-			forceDisconnectOnPingFailure := false
 			if difference > sleepDetectionInterval {
-				forceDisconnectOnPingFailure = true
 				lastTimeExecuted = w.timesource.Now()
 				w.log.Warn("keep alive hasnt been executed recently. Killing connections to peers if ping fails")
```

Comment:
We are no longer waiting for ping to fail, looks like we are closing peers just based on last ping execution. 
Hoping that is still fine. If so, we have to modify the log.